### PR TITLE
Fix redundancy issue in Terms of Use & Privacy Policy page on mobile (bug 1180428)

### PR DESCRIPTION
**Pull request by atiqueahmedziad**

### CODE REVIEW

**Comment by spasovski** [comment]
This is the wrong approach since desktop would lose its only heading. I'd simply hide the element by default and show it on `$base-tablet` via a media query.

**Comment by atiqueahmedziad** [comment]
@spasovski  done. 
pls check :)

File: src/media/css/site.styl [hunk 1]
```diff
@@ -257,6 +257,12 @@ a:active {
     }    
 }
 
+@media $wide-tablet {
+    .subheader.hidden {
```
Comment (spasovski) [inline src/media/css/site.styl]: This is the right approach but we shouldn't be calling something hidden yet give it instructions to show. It's fine to just hide the subheader on this page (try [data-page-type~=...]). Also I get the feeling you'll need to trigger the media query on `$base-tablet` instead. You're on the right path.

Comment (atiqueahmedziad) [inline src/media/css/site.styl]: ``[data-page-type~=...]`` I am a bit confused about it. I am seeing in different ``data-page-type`` we have used `website` , `homepage`, `detail`, `app-list` etc. But what will i use here ?

Comment (atiqueahmedziad) [inline src/media/css/site.styl]: This two images from (ubuntu) Firefox version 33.0
screenshot from 2015-07-08 03 38 33
screenshot from 2015-07-08 03 39 03

Comment (mstriemer) [inline src/media/css/site.styl]: The hub-and-spoke header is only shown when `[data-page-type~="leaf"]`. You should be able to hide `[data-page-type~="leaf"] .subheader` by default and then show it on wide tablets.

Comment (mstriemer) [inline src/media/css/site.styl]: And remove the `hidden` class.

**Comment by atiqueahmedziad** [comment]
@spasovski  @mstriemer r? pls :)

**Comment by mstriemer** [comment]
You've got some trailing whitespace in your CSS. Can you remove it, please?

File: src/media/css/header.styl [hunk 2]
```diff
@@ -229,6 +229,10 @@ body > .banners mkt-banner {
     }
 }
 
+[data-page-type~="leaf"] .subheader {
```
Comment (mstriemer) [inline src/media/css/header.styl]: Can you update these to include `[data-page-type~="leaf"] .secondary-header` as well? That should fix the Report an Issue and Report Abuse pages.

File: src/media/css/header.styl [hunk 3]
```diff
@@ -443,6 +449,15 @@ body > .banners mkt-banner {
     }
 }
 
+@media $wide-tablet {
+    [data-page-type~="leaf"] .subheader {
+        display: block;      
+    }
+    [data-page-type~="leaf"] .secondary-header {
```
Comment (mstriemer) [inline src/media/css/header.styl]: You can combine these two rules, above too.

```styl
[data-page-type~="leaf"] .subheader,
[data-page-type~="leaf"] .secondary-header {
    display: block;
}

**Comment by mstriemer** [comment]
I'm still seeing trailing whitespace. I put `.`s where the trailing whitespace is in one spot.

```styl
    [data-page-type~="leaf"] .subheader {
        display: block;......
    }

**Comment by atiqueahmedziad** [comment]
ok :)

**Comment by atiqueahmedziad** [comment]
is it alright ? :) @mstriemer

**Comment by atiqueahmedziad** [comment]
But see you have used white space here `[data-page-type~="detail"] .app-notices`
screenshot from 2015-07-10 12 22 12

**Comment by mstriemer** [comment]
Still some trailing whitespace. If you can setup your editor to show invisible characters or show trailing whitespace that might help.

**Comment by mstriemer** [comment]
Whitespace before the block is right, there should be a blank line between line 230 and 231. What we want to get rid of is the whitespace at the end of the line. If you select the text the spaces show up.

screenshot 2015-07-10 11 25 49
This is good.

screenshot 2015-07-10 11 26 10
This has whitespace at the end of the line.

**Comment by atiqueahmedziad** [comment]
ohh.  very much sorry, done :)

File: src/media/css/header.styl [hunk 4]
```diff
@@ -228,6 +228,10 @@ body > .banners mkt-banner {
         padding: 20px;
     }
 }
```
Comment (atiqueahmedziad) [inline src/media/css/header.styl]: do we need to give a 1 line space here ? @mstriemer

File: src/media/css/header.styl [hunk 5]
```diff
@@ -228,6 +228,10 @@ body > .banners mkt-banner {
...
+[data-page-type~="leaf"] .subheader,
```
Comment (atiqueahmedziad) [inline src/media/css/header.styl]: do we need to give a 1 line space after the `}` here ?

Comment (mstriemer) [inline src/media/css/header.styl]: There should be a space, yeah.

**Comment by mstriemer** [comment]
I don't get why the build is failing on this. Can you rebase with master please?

**Comment by atiqueahmedziad** [comment]
facing much problem with this branch . Sending a new pr ? :)

**Comment by mstriemer** [comment]
No need for a new pull request. You can rebase by pulling down the latest changes to master and then running `git rebase master` on this branch then push the updated commit.

**Comment by atiqueahmedziad** [comment]
okey :) ....this branch is 6 commits behind & 1 commit ahead ...I updated this branch 1 days ago ....rebase upsteam/master..... now fetching origin... & it is saying the branch is up to date :3

**Comment by atiqueahmedziad** [comment]
wait ..... :)

**Comment by atiqueahmedziad** [comment]
sent another pr with latest master #1380

**Comment by mstriemer** [comment]
Closing in favour of #1380.

**Pull request closed without merging**